core/scripts/testing/themes/dark.css

```diff
@@ -53,19 +53,19 @@
   --ion-color-dark-shade: #d7d8da;
   --ion-color-dark-tint: #f5f6f9;
 
-  --ion-color-medium: #989aa2;
-  --ion-color-medium-rgb: 152, 154, 162;
+  --ion-color-medium: #86919c;
+  --ion-color-medium-rgb: 134, 145, 156;
   --ion-color-medium-contrast: #000000;
   --ion-color-medium-contrast-rgb: 0, 0, 0;
-  --ion-color-medium-shade: #86888f;
-  --ion-color-medium-tint: #a2a4ab;
-
-  --ion-color-light: #222428;
-  --ion-color-light-rgb: 34, 36, 40;
-  --ion-color-light-contrast: #ffffff;
-  --ion-color-light-contrast-rgb: 255, 255, 255;
-  --ion-color-light-shade: #1e2023;
-  --ion-color-light-tint: #383a3e;
+  --ion-color-medium-shade: #768089;
+  --ion-color-medium-tint: #929ca6;
+
+  --ion-color-light: #959595;
```

Comment:
The `light` color in `dark` mode should actually be a dark hex value:

<img width="482" alt="Screenshot 2023-12-01 at 11 41 22 AM" src="https://github.com/ionic-team/ionic-framework/assets/6577830/1bc0e090-f8fb-4487-a6c2-c36de21f1c3b">

Resolving this will probaly also fix the issue of `light` and `medium` being too close.

Reply:
Thank you! I'll also create a test template to preview all these color changes - I should have done that originally!

Reply:
Ok this is ready for another look: https://github.com/ionic-team/ionic-framework/pull/28609/commits/c87940a29c91f3c4992b1edea5c2e3bdca9fbe92